From b7237f87667c3402b58c8a3c5f0905e2fa3ecf99 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Thu, 5 Nov 2020 21:23:22 +0100
Subject: [PATCH] gdb: disable ASLR PIE

As current workaround, let's disable ASLR PIE to have compiled package
gdb. Otherwise, it fails. We need to take a look before submitting this
to OpenWrt.

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 package/devel/gdb/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/package/devel/gdb/Makefile b/package/devel/gdb/Makefile
index 17b2bb8481..862ede9353 100644
--- a/package/devel/gdb/Makefile
+++ b/package/devel/gdb/Makefile
@@ -19,6 +19,7 @@ PKG_BUILD_PARALLEL:=1
 PKG_INSTALL:=1
 PKG_LICENSE:=GPL-3.0+
 PKG_CPE_ID:=cpe:/a:gnu:gdb
+PKG_ASLR_PIE:=0
 
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/nls.mk
-- 
2.25.1

